src/client/_clientwindow.py

```diff
@@ -870,12 +874,26 @@ def saveWindow(self):
         util.settings.setValue("geometry", self.saveGeometry())
         util.settings.endGroup()
 
+    def show_autojoin_settings_dialog(self):
+        autojoin_channels_list = []
+        autojoin_channels_list = Settings.get('chat/auto_join_channels')
+        text_of_autojoin_settings_dialog = """
+        Autojoin channels list : """ + ';'.join(autojoin_channels_list) + """
+        Enter the list of channels you want to join, separated by ;
```

Comment:
I'd put 'autojoin at startup' for 'join' for 120% clarity.

Reply:
done